etc/teams.profile

```diff
@@ -0,0 +1,37 @@
+# Firejail profile for teams
+# Description: Official Microsoft Teams client for Linux using Electron.
+# This file is overwritten after every install/update
+# Persistent local customizations
+include teams.local
+# Persistent global definitions
+
+# TODO add these 2 paths as blacklist to disable-programs.inc
+noblacklist ${HOME}/.config/teams
+noblacklist ${HOME}/.config/Microsoft
+
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+
+mkdir ${HOME}/.config/teams
+mkdir ${HOME}/.config/Microsoft
+whitelist ${HOME}/.config/teams
+whitelist ${HOME}/.config/Microsoft
+whitelist ${DOWNLOADS}
```

Comment:
```suggestion

```

Reply:
Download is used for file sharing through Teams. Should that essential feature be just in local profile overwrites?

Reply:
`whitelist ${DOWNLOADS}` is in electron.profile

Reply:
Sorry, my bad, remove that line from the profile asap